src/components/VaccineProgressBar/VaccineProgressBar.tsx

```diff
@@ -2,15 +2,14 @@ import React from 'react';
 import { ParentSize } from '@vx/responsive';
 import { v4 as uuidv4 } from 'uuid';
 import { ProgressBarContainer, StyledSvg } from './VaccineProgressBar.style';
-import { COLOR_MAP, VACCINATIONS_COLOR_MAP, vaccineColor } from 'common/colors';
+import { VACCINATIONS_COLOR_MAP, vaccineColor } from 'common/colors';
 import { formatPercent } from 'common/utils';
 
 export interface ProgressBarProps {
   // TODO(michael): Remove once we migrate the location page to the new version.
```

Comment:
Looks like you removed the code for `oldVersion`.  Can you remove the prop entirely?